backend/btrixcloud/pages.py

```diff
@@ -104,7 +114,13 @@ async def add_crawl_pages_to_db_from_wacz(self, crawl_id: str, batch_size=100):
                     non_seed_count += 1
 
                 if len(pages_buffer) > batch_size:
-                    await self._add_pages_to_db(crawl_id, pages_buffer)
+                    try:
+                        await self._add_pages_to_db(
+                            crawl_id, pages_buffer, ordered=False
+                        )
+                    # pylint: disable=broad-exception-caught
+                    except Exception as e:
+                        print("Error inserting, probably dupe", e)
```

Comment:
This might be a bit overly broad, especially since we're clearing the buffer even if there's an exception. Is there a chance the batch might not be written to the database that we should be handling here?